migrations/1_create_users_projects_files.js

```diff
@@ -10,6 +10,7 @@ exports.up = function (knex) {
       t.text('title').notNullable();
       t.text('tags');
       t.text('description');
+      t.text('publish_url');
```

Comment:
Can you add this to the project fixtures/tests? It might be worth me taking you through how they work.